staging/src/k8s.io/kubectl/pkg/cmd/cp/cp_test.go

```diff
@@ -307,13 +306,13 @@ func TestTarUntar(t *testing.T) {
 		},
 		{
 			name:     "relative_to_dest",
-			data:     path.Join(dir2, "foo"),
```

Comment:
are these mimicking the data that would come from a tar file? if so, shouldn't we duplicate the testcases to continue exercising "/" (which is the standard tar format) and add testcases with "\" (which is the windows-targeted case this PR is fixing)?

Reply:
I've dug into it, and there's a problem that each OS treats the other separators differently so we can't have reliable test. Iow.:
- For linux `\` is valid character that can be part of the name, so `my\dir` creates a single directory named just that. For `/` you'll get `dir` inside `my`, which is expected.
- For windows either approach `my/dir` or `my\dir` will create `dir` inside `my`. 
This means I can't reliable write windows test and execute this on linux and in the other direction when using `filepath` module which handles that for us. We'd need to manually handle linux and windows separately. So for this particular one we'll have to rely on manual testing unfortunately :disappointed:

Reply:
that's sort of why I was wondering if we should drop both `/` and `\` characters